### PR TITLE
Fix scan-app-data conversion #34283

### LINKED ISSUE #34283: [Bug]: scan-app-data conversion
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] Nextcloud Server **is** running on 64bit capable CPU, PHP and OS.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

After running occ files:scan-app-data on server v24.0.5, I'm getting 

In ScanAppData.php line 242:
Implicit conversion from float 0.4666666666666667 to int loses precision 

### Steps to reproduce

1. Install nextcloud
2. run occ files:scan-app-data
3. get error message

### Expected behavior

Not getting error message 

### Installation method

Community Manual installation with Archive

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.1

### Web server

Nginx

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 22.2.3 to 23.0.1)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

_No response_

### List of activated Apps

```shell
Enabled:
  - accessibility: 1.10.0
  - activity: 2.16.0
  - admin_audit: 1.14.0
  - bruteforcesettings: 2.4.0
  - cloud_federation_api: 1.7.0
  - comments: 1.14.0
  - dav: 1.22.0
  - federatedfilesharing: 1.14.0
  - files: 1.19.0
  - files_downloadactivity: 1.13.0
  - files_sharing: 1.16.2
  - files_trashbin: 1.14.0
  - lookup_server_connector: 1.12.0
  - provisioning_api: 1.14.0
  - serverinfo: 1.14.0
  - settings: 1.6.0
  - theming: 1.15.0
  - viewer: 1.8.0
  - workflowengine: 2.6.0
```

### Nextcloud Signing status

_No response_

### Nextcloud Logs

_No response_

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by mgoeppl**
Fixes #34283

### CODE REVIEW

**Comment by kesselb** [comment]
Hey :wave: 

Thanks for your pull request.

Should work, but our static code analysis will complain about it: https://psalm.dev/r/6dface78cb

round always returns a float and intdiv expects an integer.

File: apps/files/lib/Command/ScanAppData.php [hunk 1]
```diff
@@ -239,7 +239,7 @@ protected function showSummary($headers, $rows, OutputInterface $output) {
 	protected function formatExecTime() {
 		$secs = round($this->execTime);
```
Comment (kesselb) [inline apps/files/lib/Command/ScanAppData.php:240]:
```suggestion
		$secs = (int)round($this->execTime);
```

To make psalm happy :elephant:

**Comment by szaimen** [comment]
@mgoeppl any update here? :)

**Comment by st3iny** [comment]
Supeseded by https://github.com/nextcloud/server/pull/35935.

**Pull request closed without merging**